Os/Baremetal/TaskRunner/BareTaskHandle.hpp

```diff
@@ -19,7 +19,7 @@ class BareTaskHandle {
         bool m_enabled;
         //!< Save the priority
         NATIVE_INT_TYPE m_priority;
-        //!< Function passed in to the task
+        //!< Function passed into the task
```

Comment:
Here, `into` is the only fitting preposition.

Reply:
Agreed